diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt

```diff
@@ -549,4 +549,24 @@ class LocalVariablesWarnTest : LintTestBase(::LocalVariablesRule) {
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(WarningNames.LOCAL_VARIABLE_EARLY_DECLARATION)
+    fun `should not trigger on triple quoted strings`() {
+        val spacesForClassIndent = " ".repeat(18)
+        val spacesForFunIndent = " ".repeat(20)
+        val spacesForEndIndent = " ".repeat(16)
+        lintMethod(
+                """
+                    |class Example {
+                    |    fun some() {
+                    |       val code = ${"\"\"\"  \n${spacesForClassIndent}class Some {" +
```

Comment:
Why do you need explicit `\n` and spaces here? Why not write everything inside a string template? You can just surround triple quotes in curly braces, if that's why you did it.

Reply:
Yeah, that seems better